serverless/tensorflow/mediapipe/nuclio/main.py

```diff
@@ -0,0 +1,80 @@
+import mediapipe as mp
+from mediapipe.tasks import python
+from mediapipe.tasks.python import vision
+import json
+import base64
+import io
+from PIL import Image
+import yaml
+import numpy as np
+def init_context(context):
+    context.logger.info("Init detector...")
+
+    base_options = python.BaseOptions(
+    model_asset_path='pose_landmarker_heavy.task'
+    )
+    options = vision.PoseLandmarkerOptions(
+        base_options=base_options,
+        num_poses = 2,
+        min_pose_detection_confidence = 0.5,
+        min_pose_presence_confidence = 0.5,
```

Comment:
I am not acquainted with mediapipe, but how these parameters used? Why are there 0.5. We have also default threshold 0.55. Can the model return points with presence < 0.5 after detection? 

What if threshold, defined in runtime, after detector initialized less then 0.5, will the detector return these points as not outside?

Reply:
That 0.5 value i get from documentation example
**min_pose_detection_confidence**  - The minimum confidence score for the pose detection to be considered successful.
**min_pose_presence_confidence**  - The minimum confidence score of pose presence score in the pose landmark detection.